npm-requirements.txt

```diff
@@ -1,7 +1,4 @@
-remark@13.0.0
 remark-cli@9.0.0
-remark-lint@8.0.0
 remark-preset-lint-recommended@5.0.0
 remark-preset-lint-markdown-style-guide@4.0.0
-remark-lint-list-item-indent@2.0.0
```

Comment:
this is already included in [remark-preset-lint-recommended](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-preset-lint-recommended), so no separate install necessary.